sormas-app/app/src/main/res/layout/fragment_pathogen_test_edit_layout.xml

```diff
@@ -80,6 +85,13 @@
                     app:value="@={data.testedDisease}"
                     style="@style/ControlSingleColumnStyle" />
 
+                <de.symeda.sormas.app.component.controls.ControlSpinnerField
+                    android:id="@+id/pathogenTest_testedDiseaseVariant"
+                    app:dependencyParentField="@{pathogenTestTestedDisease}"
```

Comment:
Don't limit the visibility of this field to only CORONAVIRUS; should be visible for every disease with disease variants